### PR TITLE
docs(import): Add info about importing localized fields

### LINKED ISSUE #173: Add warning that import might remove other languages
When you Import one or more specific languages, previously created values in languages that are not in the import file will be overwritten with empty.

We should add a warning or hint to the readme.

This issue is related to https://github.com/commercetools/express-impex/issues/278

### PR DESCRIPTION
**Pull request by junajan**
#### Summary
Resolves #173

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,8 @@ The import command allows to create new products with their variants as well to
 During update it is possible to concentrate only on those attributes that should be updated.
 This means that the CSV may contain only those columns that contain changed values.
 
+**NOTE:** When importing [LocalizedString](http://dev.commercetools.com/http-api-types.html#localizedstring) fields (eg. name, description, etc.) there has to be provided all languages during the import or the missing languages will get deleted.
```
Comment (wizzy25) [inline README.md]: Just felling nitpicky 😃
`...etc.), all languages have to be provided during the import or the missing languages will be deleted.`

Comment (junajan) [inline README.md]: thx 👍 this looks better

Comment (junajan) [inline README.md]: updated e97bb88

Comment (wizzy25) [inline README.md]: 👍

**Pull request merged**